src/subscription_manager/managercli.py

```diff
@@ -1067,6 +1069,34 @@ def _do_command(self):
         else:
             self._toggle(self.options.enable or False)
 
+    def _short_description(self):
+        return _("Set if subscriptions are attached on a schedule (default of daily)")
```

Comment:
healed instead of attached?

Reply:
Good point. I also updated some other help strings.